frontend/pages/dataset/_id/annotation-mode/index.vue

```diff
@@ -53,7 +53,7 @@ export default {
   },
   data: () => {
     return {
-      areResponsesUntouched: false,
+      areResponsesUntouched: true, // NOTE - this flag is used to show or to not show a toast when questionnaire is touched (to prevent loosing current modification)
```

Comment:
We need to set the same initialization value for other components using this logic (PaginationFeedback..., )

Reply:
I add "TODO" comment to tackle this in another moment